package.json

```diff
@@ -27,6 +27,7 @@
     "@babel/core": "7.5.4",
     "babelrc-rollup": "3.0.0",
     "backstopjs": "5.0.1",
+    "cssnano": "^4.1.10",
```

Comment:
Let's do absolute versions for our packages.

Reply:
yeah and its done !!